src/gen_dragonfruit/gd_dev_apogee_sp215.cpp

```diff
@@ -4,13 +4,13 @@ void gd_dev_apogee_sp215_open(void){
     Wire.begin(9600);
 }
 
-int gd_dev_apogee_sp215_read(void){
-    int value = 555;
+uint32_t gd_dev_apogee_sp215_read(void){
+    uint32_t value = 555;
     #ifndef SEN_STUB
     Wire.beginTransmission(_PIN_GD_APOGEE_SP215_);
 
-    /* Options */
-    Wire.write(0x80);
+    /* Enable ready-bit and select 16-bit sample rate */
+    Wire.write(0x88);
```

Comment:
Since this function initializes the device, it should probably go into the open function instead.